### PR TITLE
Limiting the number of participants

### LINKED ISSUE #135: As the master of the event I want to limit the number of participants so I can manage the access to free places at the event

### PR DESCRIPTION
**Pull request by Anastasiagumenuk9**
Close #135

### CODE REVIEW

File: EventsExpress.Db/Migrations/20201007085358_AddPropIpAddress.cs [hunk 1]
```diff
@@ -1,31 +0,0 @@
-﻿using Microsoft.EntityFrameworkCore.Migrations;
```
Comment (iuriikhrystiuk) [inline EventsExpress.Db/Migrations/20201007085358_AddPropIpAddress.cs]: Why was this file removed?

File: EventsExpress.Db/Migrations/20201027164937_AddPropMaxParticipants.Designer.cs [hunk 2]
```diff
@@ -10,8 +10,8 @@
 namespace EventsExpress.Db.Migrations
 {
     [DbContext(typeof(AppDbContext))]
-    [Migration("20201007085358_AddPropIpAddress")]
-    partial class AddPropIpAddress
+    [Migration("20201027164937_AddPropMaxParticipants")]
+    partial class AddPropMaxParticipants
```
Comment (iuriikhrystiuk) [inline EventsExpress.Db/Migrations/20201027164937_AddPropMaxParticipants.Designer.cs:14]: You should not rename old migrations but add new migrations.

File: EventsExpress/ClientApp/src/components/helpers/helpers.js [hunk 3]
```diff
@@ -18,253 +18,257 @@ import Input from '@material-ui/core/Input';
...
+    ]
+
+    requiredFields.forEach(field => {
+        if (!values[field]) {
+            errors[field] = 'Required'
+        }
+    })
+
+    numberFields.forEach(field => {
+        if (values[field] && !/^[1-9]+$/i.test(values[field])) {
```
Comment (iuriikhrystiuk) [inline EventsExpress/ClientApp/src/components/helpers/helpers.js]: This regex does not allow inputs of 10, 20, 30, 100, etc. Please consider not using regex, but actually using numeric html input.

**Review by iuriikhrystiuk — CHANGES_REQUESTED** [review]

**Review by iuriikhrystiuk — APPROVED** [review]

File: EventsExpress/ClientApp/src/components/helpers/helpers.js (hunk 3 above)
Comment (Anastasiagumenuk9) [reply EventsExpress/ClientApp/src/components/helpers/helpers.js]: This part of code was deleted.
